delivery_recipe/technical-delivery/automation/README.md

```diff
@@ -0,0 +1,40 @@
+# Automation
+
+We think of automation at several different levels, from the way a developer does their work to way we build and deploy applications. 
+
+## Use better tools
+
+We encourage our developers to use the most appropriate tools for their work. Modern integrated development environments (IDEs) – like PhpStorm or Netbeans for example – not only have excellent code hinting and code completion but also lots of plugins and extras for important things, like, for instance, [PHP Code Sniffer](http://pear.php.net/package/PHP_CodeSniffer) [integration](https://gist.github.com/klederson/ca43a0f7f5b2478c1d44) [in PhpStorm](https://www.jetbrains.com/help/phpstorm/2016.2/using-php-code-sniffer-tool.html) or [Xdebug](https://xdebug.org/) [integration in PHPStorm](https://www.jetbrains.com/help/phpstorm/2016.2/configuring-xdebug.html).
+
+## Task runners and dependency managers
+
+There are many [task runners](https://www.smashingmagazine.com/2016/06/harness-machines-productive-task-runners/) and [packager or dependency managers](https://github.com/showcases/package-managers) that both help to automate the developer's work _and_ aid in conforming to industry standards and best practices.
+
+#### Frontend
+Task runners such as [Grunt](http://gruntjs.com/), [Gulp](http://gulpjs.com/) are important and useful frontend development tools, for tasks such as: 
+
+- Compiling SASS
+- Minifying SASS
+- Minifying JS
+- Concatenating/Uglifying JS
+- Compressing images
+- Watching for changes and running certain tasks on file change
+- Refreshing the browser on file change
+
+#### Backend
+
+We advocate PHP development using the [Composer](https://getcomposer.org/) dependency manager as much as possible. Composer is becoming increasingly-well supported by open-source PHP software projects and the de facto standard for building PHP software distributions and projects.
+  
+Other similar dependency managers are available and should be employed for other languages and technologies. 
+
+## Continuous integration
```

Comment:
Is there a point here where we specifically mention that the automated tasks should be run on the server? This is a really useful feature, so you don't have to commit compiled code and dependancies.